lib/platform/bitbucket-server/index.md

```diff
@@ -53,3 +52,11 @@ You should then receive a "Configure Renovate" onboarding PR in any projects tha
 
 We support all Bitbucket Server versions which are not EOL.
 See [Atlassian Support End of Life Policy](https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html#AtlassianSupportEndofLifePolicy-BitbucketServer) for uptodate versions.
+
+## Assignees support
+
+Assignee support was added in version `17.2.0` of Renovate.
+It's available in all Bitbucket versions supported by Renovate.
+
+**Note:** Use username slug as assignee name.
+Which can be easy found here: `https://{bitbucket_server}/rest/api/1.0/users`
```

Comment:
This `{bitbucket_server}` thing in the URL looks very weird.

I'd like you to replace `https://{bitbucket_server}/rest/api/1.0/users` with a proper link, if possible.

Reply:
Problem is that is self hosted solution. For every user of bitbucket-server has other url. Maybe I can only provide path to the resource.

```suggestion
Which you can found on your bitbucket instance on `/rest/api/1.0/users` for every user.
```

Reply:
Trying to understand how this works...

If you're running a Bitbucket instance, you can use the Bitbucket API to fetch a list of users on that instance? You can use `https://{bitbucket_server}/rest/api/1.0/users` where you replace `{bitbucket_server}` to get a list of user slugs?

Reply:
You get lijst of users which include the slug. The problem is that the slug is not used very common in bitbucket-server application. So I thought to help others where the can find easily this information for every user the want to add to the list

Reply:
Ah okay, the `slug` is not used very often, so people probably don't know where to even find that `slug`. So that's why you want to explain how to find the slug in our documentation.

I'd like to have that sentence in our docs to help our readers. But it will need to be improved, right now it's not really a proper English sentence.

How about something like this:

```suggestion
Fire a request at `https://{bitbucket_server}/rest/api/1.0/users` to get a list of Bitbucket users which includes the slugs.
```

I'm not sure what the correct verb is (doing-things word) to describe the action that the user performs on the URL. You programmer types probably have a word for this... 😉

Reply:
Yes, and I like your suggestion. That explain correctly what the can do to get slugs of the users. Fire is a word I also would use in this sentence, I think. But I'm not native speaker.

Reply:
Is there any way to get the slug through the Bitbucket Server UI? Any documentation from Atlassian which we can link to as well?

Reply:
If you go to profile page , it is in the url: `https://{bitbucket_server}/users/${slug}`
On profile page itself is username which is mostly same as te slug, but can be different because that are two different fields in the user API.

I found this technical documentation of user slug: https://docs.atlassian.com/bitbucket-server/javadoc/5.0.0/api/reference/com/atlassian/bitbucket/user/ApplicationUser.html , but not most useful I think

Reply:
So perhaps we can say that you can test if a username is the same as the slug by trying URL X ?

Reply:
Something like this?

**Note:** Use username slug as assignee name.
Slug is most of the time the same as the username.
Fire a request at `https://{bitbucket_server}/rest/api/1.0/users` to get a list of Bitbucket users which includes the slugs.
You can check this by checking if the profile page exists.
Other option: If you go to the profile page of a user, then the slug is in the url.
(e.g. `https://{bitbucket_server}/users/{username}`)

Reply:
Sorry, I was confused between Reviewers and Assignees. Assignees is not supported in bitbucket server.  I leave to small fixes in documentation in this PR.